### PR TITLE
[Core][GeometryData] Enable serializer for GeometryData

### PR DESCRIPTION
**Pull request by tteschemacher**
Hi all,

I want to discuss here the bugfix to parse the geometry data, which is needed as long as this object is non-static, which is the case for e.g. quadrature points.

Please also comment which checks should be added?

Thanks!

### CODE REVIEW

File: kratos/geometries/geometry_shape_function_container.h [hunk 1]
```diff
@@ -379,7 +381,8 @@ class GeometryShapeFunctionContainer
 
     virtual void load( Serializer& rSerializer )
     {
-        KRATOS_ERROR << "load function for geometry_shape_function_container not yet implemented." << std::endl;
+        rSerializer.save("default_method", default_method);
```
Comment (jcotela) [inline kratos/geometries/geometry_shape_function_container.h]:
```suggestion
        rSerializer.load("default_method", default_method);
```

**Review by jcotela — DISMISSED** [review]
Minor but important change, I think...

**Comment by tteschemacher** [comment]
Has someone a glue where the travis error massage comes from?

**Comment by philbucher** [comment]
you should merge master then it is gone

**Comment by tteschemacher** [comment]
thx

File: kratos/geometries/geometry_shape_function_container.h (hunk 1 above)
Comment (tteschemacher) [reply kratos/geometries/geometry_shape_function_container.h]: thanks, this is of course a bit crucial... ;)

**Comment by tteschemacher** [comment]
closing this PR as this is resolved now by #7473

**Pull request closed without merging**